packages/ember-template-compiler/lib/plugins/transform-attrs-into-args.js

```diff
@@ -27,6 +27,20 @@ export default function TransformAttrsToProps() {
   this.syntax = null;
 }
 
+function isAttrs(node) {
+  if (node.parts[0] === 'attrs') {
+    return true;
+  }
+
+  let [_this, attrs] = node.parts;
+
+  if (_this === null && attrs === 'attrs') {
+    node.parts.shift();
+    node.original = node.original.slice(5);
+    return true;
+  }
```

Comment:
We said we weren't going to introduce any deprecations as part of 2.9.0. We should deprecate a lot of this stuff in 2.10.0.